homeassistant/components/ozw/__init__.py

```diff
@@ -357,6 +359,7 @@ def async_handle_scene_activated(hass: HomeAssistant, scene_value: OZWValue):
         {
             const.ATTR_NODE_ID: node_id,
             const.ATTR_SCENE_ID: scene_id,
+            const.ATTR_SCENE_INSTANCE: ozw_instance,
```

Comment:
Should we change here too?

Reply:
Sure thing, should I adjust the order at all?